frontend/src/component/releases/ReleasePlanTemplate/TemplateForm/MilestoneList/MilestoneCard/MilestoneCard.tsx

```diff
@@ -113,9 +115,10 @@ const StyledIconButton = styled(IconButton)(({ theme }) => ({
 
 const StyledDragIcon = styled(IconButton)(({ theme }) => ({
     padding: 0,
+    position: 'absolute',
```

Comment:
I'm making this icon absolutely positioned in the first iteration, but it has some issues when we decrease padding on smaller screens, so I'll probably turn each milestone into a full-width component that handles its own padding before the icon and the following card.